.github/workflows/live/afp-ids.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
```

Comment:
Could we get a one liner description of what this script does?

Reply:
added